content/en/docs/components/model-registry/getting-started.md

```diff
@@ -0,0 +1,170 @@
++++
+title = "Getting started"
+description = "Getting started with Model Registry using examples"
+weight = 30
+
++++
+
+This guide shows how to get started with Model Registry and run a few examples using the
+command line or Python clients.
+
+At this time, there is not yet an UI for Model Registry; therefore, this documentation focus on backend services and APIs.
+
+For an overview of the logical model of model registry, check the
+[Model Registry logical model](https://github.com/kubeflow/model-registry/blob/main/docs/logical_model.md).
+The logical model is exposed via the Model Registry [REST API](https://editor.swagger.io/?url=https://raw.githubusercontent.com/kubeflow/model-registry/main/api/openapi/model-registry.yaml).
+
+## Prerequisites
+
+To follow along the examples in this guide, you will need a Kubeflow installation and the Model Registry installed:
+
+- Kubeflow [installed](docs/started/installing-kubeflow/)
+- Model Registry [installed](/docs/components/model-registry/installation/)
```

Comment:
the link is not working for me

Reply:
I forgot a `/`, thank you Josh, fixed with https://github.com/kubeflow/website/pull/3698/commits/d9c3cd6ed939dc22de87671489be976f10756b82

![image](https://github.com/kubeflow/website/assets/1699252/df2f2aed-ab14-428d-9e5c-1503df21eb16)